Specifications/QIR/Data-Types.md

```diff
@@ -2,15 +2,11 @@
 
 We define LLVM representations for a variety of classical and quantum data types.
 
-In most cases, we represent complex types as pointers to opaque LLVM structure
-types.
-This allows each target to provide a structure definition appropriate for that
-target.
+Representing the types used for qubits and measurement results as pointers to opaque LLVM structure types allows each target to provide a structure definition appropriate for that target.
```

Comment:
```suggestion
Representing the types used for qubits and measurement results as pointers to
opaque LLVM structure types allows each target to provide a structure definition
appropriate for that target.
```

A minor point: keeping the line length limited makes it easier to read diffs (PRs).